helm/wfl/Chart.yaml

```diff
@@ -4,7 +4,7 @@ description: A Helm chart for deploy WorkFlow Launcher on Kubernetes
 
 type: application
 
-version: 0.3.36
+version: 0.3.37
```

Comment:
For my records, when we're ready to deploy WFL to dev and/or prod we'll bump the version in `gotc-deploy`:

https://github.com/broadinstitute/gotc-deploy/blob/main/deploy/gotc-dev/helm/helmfile.yaml#L892
https://github.com/broadinstitute/gotc-deploy/blob/main/deploy/gotc-prod/helm/helmfile.yaml#L701